cfgov/jinja2/v1/story-page/index.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'layout-2-1-bleedbar.html' %}
+
+{% import 'macros/video-player.html' as video_player %}
+{% import 'organisms/item-introduction.html' as item_introduction with context %}
+{% import 'templates/render_block.html' as render_block with context %}
+{% import 'templates/streamfield-sidefoot.html' as streamfield_sidefoot with context %}
+
+{% block content_main_modifiers -%}
+    {{ super() }} content__flush-bottom
```

Comment:
Yes-- Jinja doesn't automatically call bare functions (ie, it behaves more like Python)